spras/evaluation.py

```diff
@@ -67,63 +67,73 @@ def load_files_from_dict(self, gold_standard_dict: Dict):
 
         node_table is a single column of nodes pandas table.
-
-        returns: none
         """
         self.label = gold_standard_dict['label']  # cannot be empty, will break with a NoneType exception
         self.datasets = gold_standard_dict['dataset_labels']  # can be empty, snakemake will not run evaluation due to dataset_gold_standard_pairs in snakemake file
 
         data_loc = gold_standard_dict['data_dir']
 
         # cannot be empty, snakemake will run evaluation even if empty
-        node_file = gold_standard_dict["node_file"]
-        edge_file = gold_standard_dict["edge_file"]
+        node_files = gold_standard_dict["node_files"]
+        edge_files = gold_standard_dict["edge_files"]
 
         # exactly one gold standard file kind can be present in a gold standard dataset
-        has_node_file = len(node_file) > 0
-        has_edge_file = len(edge_file) > 0
+        has_node_files = len(node_files) > 0
+        has_edge_files = len(edge_files) > 0
 
-        if has_node_file and has_edge_file:
+        if has_node_files and has_edge_files:
             raise ValueError(
-                f"Gold standard '{self.label}': both node_file and edge_file provided. "
-                "Exactly one is allowed in a gold standard dataset."
+                f"Gold standard '{self.label}': both node_files and edge_files provided. "
+                "Only one type is allowed in a gold standard dataset."
             )
-        if not has_node_file and not has_edge_file:
+        if not has_node_files and not has_edge_files:
             raise ValueError(
-                f"Gold standard '{self.label}': neither node_file nor edge_file provided."
+                f"Gold standard '{self.label}': neither node_files nor edge_files provided."
             )
 
-        # TODO: later iteration - allow for multiple node or edge files in the list
-        # The files will be merged together to make one large node or edge table
-        if has_node_file and len(node_file) != 1:
-            raise ValueError(
-                f"Gold standard '{self.label}': node_file must contain exactly one file."
-            )
-        if has_edge_file and len(edge_file) != 1:
-            raise ValueError(
-                f"Gold standard '{self.label}': edge_file must contain exactly one file."
-            )
+        if has_node_files:
 
-        if has_node_file:
-            single_node_table = pd.read_table(os.path.join(data_loc, node_file[0]), header=None)
-            if single_node_table.shape[1] != 1:
-                raise ValueError(
-                    f"Gold standard '{self.label}': the provided node_file must have exactly 1 column of nodes."
-                )
-            single_node_table.columns = [self.NODE_ID]
-            self.node_table = single_node_table
+            node_tables = []
+            for node_file in node_files:
+                node_table = pd.read_table(os.path.join(data_loc, node_file), header=None)
```

Comment:
No header is a different format than our input files. Is that an issue? It should be consistent with whatever we end up doing for the new indicator files in #254.

Reply:
For the nodes, I agree we should put the header NodeID to stay consistent. I'm not sure what we should put for edge gold standards since for edge files in datasets, there is no header. I was assuming we could do Interactor1, Interactor2, (optional) Direction as the header.